<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>651</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_low">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>71</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_high">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_eps">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>120</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_func">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>170</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>470</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Метод Симпсона</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>360</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Метод средних прямоугольников</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>270</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Ответ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>580</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Метод Гаусса 4-го порядка</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1043</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
